templates/quickstart-bastion-for-atlassian-services.yaml

```diff
@@ -50,7 +50,7 @@ Parameters:
 Resources:
   Bastion:
     Type: AWS::EC2::Instance
-    DeletionPolicy: Retain
+    DeletionPolicy: Delete
     UpdateReplacePolicy: Retain
```

Comment:
Ok, so when there are changes in the parent stack - e.g. number of nodes, it won't try to replace the bastion host.
How does it behave if you change `BastionHostRequired` in the product-with-vpc stack from `true` to `false`? Does it remove or retain the EC2 instance?

Reply:
Good question. I can test that.